.github/workflows/packaging.yml

```diff
@@ -0,0 +1,93 @@
+name: packaging
+
+on:
+  pull_request:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - '*'
+
+jobs:
+  package:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - { os: 'debian', dist: 'stretch' }
+          - { os: 'debian', dist: 'buster' }
+          - { os: 'debian', dist: 'bullseye' }
+          - { os: 'el', dist: '7' }
+          - { os: 'el', dist: '8' }
+          - { os: 'fedora', dist: '30' }
+          - { os: 'fedora', dist: '31' }
+          - { os: 'fedora', dist: '32' }
+          - { os: 'fedora', dist: '33' }
+          - { os: 'fedora', dist: '34' }
+          - { os: 'fedora', dist: '35' }
+          - { os: 'fedora', dist: '36' }
+          - { os: 'ubuntu', dist: 'xenial' }
+          - { os: 'ubuntu', dist: 'bionic' }
+          - { os: 'ubuntu', dist: 'focal' }
+          - { os: 'ubuntu', dist: 'groovy' }
+          - { os: 'ubuntu', dist: 'hirsute' }
```

Comment:
CI is red here.

Reply:
![image](https://user-images.githubusercontent.com/57662928/180781661-58827253-048a-420e-b00c-bb13abe480cf.png)
https://github.com/tarantool/queue/commit/8f5cc970aba0d1ab5b4858fab767ad297275daee#diff-b262e6f808c6e8d409cdeeeec179bcf85a312d8ad9e16b8c7861dbe538b62b0b

Reply:
These fails caused by disabling Ubuntu repositories of EOL hisrute and impish. @0x501D, can I remove these OS versions from the workflow?

Reply:
Yes, I see no reason to run CI on EOL distributions